plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveFaultTolerantExecutionConnectorTest.java

```diff
@@ -14,15 +14,18 @@
 package io.trino.plugin.hive;
 
 import io.trino.testing.FaultTolerantExecutionConnectorTestHelper;
+import io.trino.testing.QueryRunner;
 
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 public class TestHiveFaultTolerantExecutionConnectorTest
         extends BaseHiveConnectorTest
 {
-    public TestHiveFaultTolerantExecutionConnectorTest()
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
     {
-        super(FaultTolerantExecutionConnectorTestHelper.getExtraProperties());
+        return BaseHiveConnectorTest.createHiveQueryRunner(FaultTolerantExecutionConnectorTestHelper.getExtraProperties());
```

Comment:
nit: static import

Reply:
I would rather leave it as is to bot make it confusable with non-static `getQueryRunner` method

Reply:
Maybe make it non-static and `super.` here?

Reply:
IDK. It felt nicer to me to keep the contract that to extend the abstract the `BaseHiveConnectorTest` it is a sole responsibility of leaf class to provide implementation of `createQueryRunner`. 
In that case keeping `BaseHiveConnectorTest.createHiveQueryRunner` as static makes more sense. Then `BaseHiveConnectorTest` serves just as a namespace for helper `createHiveQueryRunner` to be used by the interested parties. The fact it is part of the hierarchy does not really matter.

If you do not feel strongly I would keep it.